scala-fx/src/main/scala/fx/Nullable.scala

```diff
@@ -0,0 +1,86 @@
+package fx
+
+/**
+ * An unwrapped option type. Allows for nullability handling without wrapping the nullable
+ * value.
+ */
+opaque type Nullable[A] = A | Null
```

Comment:
nice addition

Reply:
Should be in the scala 3 std lib, to be honest.